pom.xml

```diff
@@ -467,6 +467,7 @@ Additionally it includes CPD, the copy-paste-detector. CPD finds duplicated code
                                 <version>${ant.version}</version>
                             </additionalDependency>
                         </additionalDependencies>
+                        <excludePackageNames>*.internal</excludePackageNames>
```

Comment:
Could we also exclude the packages under `lang.*.rule.*`? IMO the rule implementations should be internal, their Javadoc is irrelevant